supertokens_python/recipe/emailverification/emaildelivery/services/backward_compatibility/__init__.py

```diff
@@ -18,21 +18,29 @@
 from httpx import AsyncClient
 from supertokens_python.ingredients.emaildelivery.types import \
     EmailDeliveryInterface
+from supertokens_python.logger import log_debug_message
 from supertokens_python.recipe.emailverification.interfaces import \
     TypeEmailVerificationEmailDeliveryInput
 from supertokens_python.recipe.emailverification.types import User
 from supertokens_python.supertokens import AppInfo
+from supertokens_python.utils import handle_httpx_client_exceptions
 
 
 def default_create_and_send_custom_email(app_info: AppInfo) -> Callable[[User, str, Dict[str, Any]], Awaitable[None]]:
     async def func(user: User, email_verification_url: str, _: Dict[str, Any]):
         if ('SUPERTOKENS_ENV' in environ) and (environ['SUPERTOKENS_ENV'] == 'testing'):
             return
+        data = {}
         try:
+            data = {'email': user.email, 'appName': app_info.app_name, 'emailVerifyURL': email_verification_url}
```

Comment:
```suggestion
        data = {'email': user.email, 'appName': app_info.app_name, 'emailVerifyURL': email_verification_url}
        try:
```

Reply:
We are passing `data` to `handle_httpx_client_exceptions` outside try/except block. So it needs to be defined beforehand.

Reply:
See my comment again. It's above the try block